src/tools/rbd_nbd/rbd-nbd.cc

```diff
@@ -803,8 +803,10 @@ static int rbd_nbd(int argc, const char *argv[])
       cmd = Connect;
     } else if (strcmp(*args.begin(), "unmap") == 0) {
       cmd = Disconnect;
+      g_ceph_context->_conf->set_val_or_die("admin_socket", "");
```

Comment:
@liupan1111 I don't really think there is any pressing requirement for invoking "global_init" before processing the command line. Let me know if that assumption is incorrect.